app/apps/addresses/views.py

```diff
@@ -48,28 +48,27 @@ class AddressViewSet(ViewSet, GenericAPIView, PermitDetailsMixin):
     )
     def residents_by_bag_id(self, request, bag_id):
         address = self.queryset.filter(bag_id=bag_id).first()
-        if not address:
```

Comment:
ipv `self.queryset` hoor je `self.get_queryset()` te gebruiken (heeft met performance te maken).
Nog beter is om de built-in van deze class self.get_object() te gebruiken.